docs/pages/taglines.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Taglines
+layout: variation
+section: patterns
+status: Released
+description: >-
+  Taglines are short paragraphs of text with a USA flag to their left.
+variation_groups:
+  - variations:
+      - variation_name: "Standard tagline"
+        variation_code_snippet: >-
+          <div class="a-tagline">
+              <span class="u-usa-flag"></span>
+              <div>
+                An official website of the
+                <span class="a-tagline_usa">United States government</span>
+              </div>
+          </div>
+        variation_description: ""
+        variation_implementation: >-
+          The flag itself is a stand-alone element of
+          `<span class="u-usa-flag"></span>`
+          that uses a utility class that embeds a double-resolution flag png
+          via a data URI.
+      - variation_name: "Large tagline"
+        variation_code_snippet: >-
+          <div class="a-tagline a-tagline__large">
+              <span class="u-usa-flag"></span>
+              <div>
+                An official website of the
```

Comment:
Totally trivial but it's annoying how netlify CMS seems to mix two- and four-space indentations.